### ISSUE TITLE
Add .htaccess support.

### ISSUE THREAD

**Issue by emmihr** [kzahel/web-server-chrome#191]
Hello! I don't know if this is already implemented and I'm doing it wrong, but when I put an .htaccess file in my server's root folder, containing a simple
    ErrorDocument /404.html
line, and then go to a non-existent page on the server, such as
    127.0.0.1:8887/jgjguthudg.html
I still get an Entry Not Found. Is there already a default htaccess I can edit / a way to add in or otherwise emulate an htaccess file?

**Comment by ecc521** [comment]
It's my belief that this is supposed to be a relatively basic web server, and not the kind of thing that one would want to host any kind of production website on. Given the fact that this is a chrome extension, and therefore can't run native code, one could expect absolutely horrible performance if it tried to behave like a fully-featured web server.

If you are looking to use .htaccess, you should use a local Apache web server (after all, .htaccess is an Apache configuration file). If all you want is custom 404 page support for this extension, look at issue #182.

**Comment by emmihr** [comment]
Right, but just because I need, say, a custom 403 page as well, how
would that work? htaccess doesn't need to be mandatory. I'm not able to run
an Apache web server because all I have is an unrooted Android O phone, and
my ARM Chromebook.

**Comment by ecc521** [comment]
Although Linux for Chromebooks doesn't interface with localhost, you can still use a server with it. Just type ```hostname -I``` to get the IP address of the Linux terminal.

**Comment by emmihr** [comment]
I'm on an older Chromebook, ARMHF, so I don't have anything like apt, whch
means that web server chrome is my best option.

**Comment by kzahel** [comment]
Quite an impressive feature Ethan!
